helios/views.py

```diff
@@ -1315,6 +1315,12 @@ def voters_email(request, election):
   if not template in [t[0] for t in TEMPLATES]:
     raise Exception("bad template")
 
+  template_tyte = ('','')
```

Comment:
maybe we just need to change the templates to be a hashmap instead of this code?